app/views/govuk_publishing_components/components/_attachment.html.erb

```diff
@@ -0,0 +1,48 @@
+<%
+  attachment = GovukPublishingComponents::Presenters::Attachment.new(attachment)
+  target ||= nil
+  hide_help_text ||= false
+  attributes = []
+
+  if attachment.content_type_abbr
+    content = tag.abbr(attachment.content_type_abbr,
+                       class: "gem-c-attachment__abbr",
+                       title: attachment.readable_content_type)
+    attributes << tag.span(content)
+  end
+
+  if attachment.readable_file_size
+    attributes << tag.span(attachment.readable_file_size)
+  end
+
+  if attachment.readable_number_of_pages
+    attributes << tag.span(attachment.readable_number_of_pages)
+  end
+%>
+<%= tag.section class: "gem-c-attachment" do %>
+  <%= tag.div class: "gem-c-attachment__thumbnail" do %>
+    <%= link_to attachment.url, target: target, tabindex: "-1" do %>
```

Comment:
Are we marking this as aria-hidden as per Whitehall: https://github.com/alphagov/whitehall/blob/3cee9551c6247c7edded9204b4f5f4ed0d71c514/app/views/documents/_attachment.html.erb#L5

Reply:
I've added `aria-hidden: true` to this link